api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -437,7 +427,9 @@ from \`${BQ_PROJECT}.${BQ_DATASET}.concept\` c inner join
 on co1.observation_source_concept_id = c.concept_id
 inner join \`${BQ_PROJECT}.${BQ_DATASET}.concept_relationship\` r
 on r.concept_id_2 = c.concept_id
-where r.concept_id_1 in(1586134, 1585855,1855710) and value_as_number is not null
+where c.vocabulary_id = 'PPI' and
+c.concept_id not in (select i.concept_id from \`${WORKBENCH_PROJECT}.${WORKBENCH_DATASET}.ignore_ppi\` i)
```

Comment:
So I don't think we want a table with concept IDs for PPI. If you want to filter out PPI, that should be possible using SQL like vocabulary_id != ‘PPI’.

That said... if the issue here is achilles doesn't work well on free text answers, there may be other observations in the data that are free text answers... we should find a way to exclude them... let's chat about this on #curation.

Reply:
Also, we may *not* want to exclude all PPI, but rather all free text PPI answers. But maintaining a list of concept IDs for that is not a good answer. We should find a way of excluding these rows in another fashion...

Reply:
We are only excluding the ppi concept_ids in ignore_ppi .. They are things like first name, address line 1 , etc.